src/main/kotlin/Graph.kt

```diff
@@ -0,0 +1,39 @@
+import Graph.Companion.STOCKHOLM
+import Graph.Companion.UPPSALA
+
+class Graph {
+    companion object {
+        const val STOCKHOLM = "ID_STOCKHOLM"
+        const val UPPSALA = "ID_UPPSALA"
+    }
+
+    fun shortestPath(node1: Node, node2: Node): Int {
+        return 0
+    }
+
+    fun getNodeById(id: String): Node {
+        return Node()
+    }
+}
+
+class Node {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tokgozmusa/detekt-danger/blob/188a4025652c68e08e9ac84e526e71fcf9c28a5a/src/main/kotlin/Graph.kt#L19"></span>The class or object Node is empty.</td>
    </tr>
  </tbody>
</table>